include/wlr/types/wlr_surface.h

```diff
@@ -59,6 +62,9 @@ struct wlr_surface_state {
 	} viewport;
 
 	struct wl_listener buffer_destroy;
+
+	size_t cached_locks;
```

Comment:
Right. Since this is part of the surface state struct, repeating "state" is a little weird, but I guess more explicit is better.

Renamed alongside with `cached_link`, which has the same issue.